CSharpSource/Source/api/Social/Manager/SocialManagerPresenceRecord.cs

```diff
@@ -63,6 +63,16 @@ public override bool Equals(object obj)
             return this.Equals((SocialManagerPresenceRecord)obj);
         }
 
+        public override int GetHashCode()
+        {
+            unchecked
+            {
+                var hashCode = UserState.GetHashCode();
+                hashCode = (hashCode * 397) ^ (m_titleRecords.GetHashCode());
```

Comment:
How did you come up with this?

Reply:
It mimics the hash codes of the other classes.